sap/adt/common_types.py

```
@@ -0,0 +1,86 @@
+    def __init__(self, total_item_count, items):
+        self.total_item_count = total_item_count
+        self.items = items
+
+    def __iter__(self):
+        return iter(self.items)
+
+    @staticmethod
+    def from_xml(xml_content):
+        """Parse namedItemList XML into a NamedItemList instance"""
+
+        root = ElementTree.fromstring(xml_content)
```

Ruff (0.15.12)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)